docs/_data/toc.yml

```
@@ -31,6 +31,8 @@ toc:
     subfolderitems:
       - page: PCP Data collection
         url: /supremm-compute-pcp.html
+      - page: Job Script collection
```

May as well change this one too:
```suggestion
      - page: Job Batch Script collection
```